README.md

```diff
@@ -19,6 +19,14 @@ https://nominatim.org/release-docs/develop/ .
 Installation
 ============
 
+**Nominatim is a complex piece of software and runs in a complex environment.
+Installing and running Nominatim is something for experienced system
+administrators only who can do some trouble-shooting themselves. We are sorry,
+but we can not provide installation support. We are all doing this in our free
+time and there is just so much of that time to go around. Do not open issues in
+our bug tracker if you need help. You can ask questions on the mailing list
+(see below) or on [help.openstreetmap.org](https://help.openstreetmap.org/).**
+
```

Comment:
As the FAQ is a bit buried in the install section, maybe add an explicit hint to it here:

"Also, be sure to thoroughly review the FAQ section in the troubleshooting guide (https://nominatim.org/release-docs/latest/admin/Faq/)."